lib/core/utils/responsive_helper.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flutter/material.dart';
+
+class ResponsiveHelper {
+  static const double tabletMinWidth = 800;
+  static const double desktopMinWidth = 1200;
+
+  static bool isMobile(BuildContext context) =>
+      MediaQuery.of(context).size.width < tabletMinWidth;
+
+  static bool isTablet(BuildContext context) =>
+      MediaQuery.of(context).size.width >= tabletMinWidth &&
+      MediaQuery.of(context).size.width < desktopMinWidth;
+
+  static bool isDesktop(BuildContext context) =>
+      MediaQuery.of(context).size.width >= desktopMinWidth;
```

Comment:
I am not sure what this does. How can a width be related to desktop?

Reply:
It checks the screen width of the device. If the width is good for our desktop width, it returns true.